js/Graph.js

```diff
@@ -436,6 +436,9 @@ export default function Graph(props) {
     <line stroke="rgba(0,0,0,.15)" stroke-width="6px" x1="9" x2="9" y2="12"/>
     </pattern>`;
 
+    // Remove background fill (so app theme shows through)
+    d3.select('#graph .graph polygon').remove();
```

Comment:
I think we should keep it so it can be exported (right?), but color it via CSS

Reply:
I don't think so.  This is just removing the background rectangle that `graphviz` adds (and fills with white). It doesn't serve any purpose when printed, and otherwise just obscures the background color of the page, which we want to show through.